### DISCUSSION TITLE
websites that I successfully built and deployed suddenly not working on Vercel

### DISCUSSION THREAD

**Discussion by stephendewyer** [vercel/community#2534] [Bug Report]
I successfully built and deployed a number of websites on Vercel that now, for some reason, no longer work.  The websites are https://suzannecontiquilts.vercel.app/, https://animatedtabpanels.vercel.app/ and https://dynamicfooter.vercel.app/.  Not sure why they stopped working.  Can anyone help?

**Comment by amyegan** [comment]
Hi @stephendewyer. These three sites seem to be working appropriately today. It looks like there was a caching issue over the weekend, but it has been resolved. You can find more details about it at https://www.vercel-status.com/incidents/mg87wqkq22nt
